src/main/java/Deadline.java

```diff
@@ -3,11 +3,20 @@
 import java.time.LocalTime;
 import java.time.format.DateTimeFormatter;
 
+/**
+ * Representation of a Deadline task. Inherits from Task.
+ */
 public class Deadline extends Task {
     String deadLine;
```

Comment:
Perhaps it'd be more suitable if this String is named something eg. description instead of deadLine?
```suggestion
    String deadLine;
```
```suggestion
    String description;
```

Reply:
I agree with @figo2127's comment! Might confuse yourself with the `Deadline` model name.

Reply:
Agree